### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by SloCompTech** [RPi-Distro/pi-gen#399]
Hi,
I have some ideas to improve documentation:
- Make **tree** for stage directory structure.
- Add (for ex. 00) at 4. point of How the build process work
- Explain available environment variables available when running script
- Explain how to run command as if you were on RPi

**Comment by janis-ax** [comment]
Yes I think it would be a good idea. (But mostly a look to the existing code will help 😄 )
